### PR TITLE
DOCSP-27767 Scriptable Compass IA Changes

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
**Scope of this ticket is to reorganize content within the `Connect to MongoDB` tab. Greater IA changes regarding the user journey through configuration file, command line, and UI settings will be made in a different ticket.**

- Consolidated content from the [Start Compass from the Command Line](https://www.mongodb.com/docs/compass/current/connect/connect-from-the-command-line/) and [Compass Command Line Options](https://www.mongodb.com/docs/compass/current/command-line-options/) pages into a CLI tab under "Connect to MongoDB"
- Cleaned up the advanced connection options under the new "Advanced Connection Options" tab.

## STAGING
- [Connect to MongoDB](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27767-scriptable-compass-ia-changes/connect/) (see TOC under here)
- NEW: [Advanced Connection Options landing page](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27767-scriptable-compass-ia-changes/connect/advanced-connection-options/) - holds advanced options task pages 
- [Start Compass from the Command Line](https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-27767-scriptable-compass-ia-changes/connect/connect-from-the-command-line/) - opted to make this a parent folder rather than putting all the information together in one page. Otherwise, the page would've had too much information all at once. 

## JIRA
https://jira.mongodb.org/browse/DOCSP-27767

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=63fe3fe305dd3cc81493a301

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/connect/advanced-connection-options.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/intro-advanced-connection-options.rst
+
+Options
+-------
+
```
Comment (sarah-olson-mongodb) [inline source/connect/advanced-connection-options.txt:19]: (proper noun?)
Just a question. Is Connection String Scheme truly a proper noun?

File: source/connect/advanced-connection-options.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+:ref:`general-connection-tab`
+  Learn how to select a Connection String Scheme for your deployment. 
+
+:ref:`authentication-connection-tab`
+  Learn how to connect deployments that require authentication.
+
```
Comment (sarah-olson-mongodb) [inline source/connect/advanced-connection-options.txt:25]: (wording)
This pertains to each of these line items.

Are you learning how to connect deployments WITH TLS/SSL?

Or...
Are you learning how to connect deployments that have TLS/SSL enabled?

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks so much @ajhuh-mdb! In terms of the IA updates, this is exactly what was needed. Great job! I made a few wording tweaks, here and there. 

In terms of future work, you might consider applying some additional information typing to this area of the docs. I know this isn't the scope of your work here, so I've omitted comments around this, but perhaps worth a follow up project or ticket?

File: source/connect.txt [hunk 3]
```diff
@@ -85,42 +85,18 @@ on GitHub.
 .. include:: /includes/steps/starting-compass-paste-string.rst
 
 Connecting with Compass from the Command Line
-----------------------------------------------
-
+---------------------------------------------
```
[the remark this answers is not in the document]
Comment (ajhuh-mdb) [reply source/connect.txt:88]: Noted and fixed!

File: source/includes/intro-advanced-connection-options.rst [hunk 4]
```diff
@@ -0,0 +1,3 @@
+The Advanced Connection Options provide additional forms of connecting Compass 
+to MongoDB. Advanced connection options allow you to specify authentication, 
+TLS/SSL, and SSH to securely connect to your deployment.
```
[the remark this answers is not in the document]
Comment (ajhuh-mdb) [reply source/includes/intro-advanced-connection-options.rst:3]: Looks like in existing documentation as well as the Compass UI, it's a proper noun. Will update accordingly.

File: source/connect/advanced-connection-options.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/connect/advanced-connection-options.txt:19]: Looks like it was mistakenly capitalized bc of the styling in the Compass UI. Fixing across docs.

File: source/connect/advanced-connection-options.txt (hunk 2 above)
Comment (ajhuh-mdb) [reply source/connect/advanced-connection-options.txt:25]: Will double check in external review, but it looks like it should all be "WITH TLS/SSL".

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @ajhuh-mdb!
LGTM

One thought is that since IA refactors can be tricky, it might be worth making a ticket for the end of the epic, once everything is completed, to spend some time re-reviewing the TOC to ensure that everything still makes sense, once the rest of the work is done.

**Comment by ajhuh-mdb** [comment]
Noted! I'll add a note to revisit this section in phase 3 of the [Compass Settings UI and Global Config File](https://docs.google.com/document/d/1ebY_1FwLnwY_oGaNRb8s3whS-VRFxE_08_iKFEERTs8/edit#) project. :)

**Pull request merged**